### ISSUE TITLE
Add a mach command to set up rust compiler for particular target

### ISSUE THREAD

**Issue by jdm** [servo/servo#23198]
This would make it easier to script the ARM build setup, which requires fetching the appropriate rust standard library for a particular target. Waiting for the build to complete before continuing is annoying.

**Comment by jschwe** [comment]
I don't think this is necessary anymore. in CI, the targets are fetched automaticallly and locally `rustup` prints a useful message on how to do the one-time setup.
